package.json

```diff
@@ -96,6 +95,7 @@
     "ts-node": "^10.9.1",
     "twrnc": "^4.5.1",
     "typescript": "~5.2.2",
+    "typescript-eslint": "^8.7.0",
```

Comment:
Bumps packages to align with core https://github.com/MetaMask/core/pull/4727/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R93-R100